<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>resultDialog</class>
 <widget class="QDialog" name="resultDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>551</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>551</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Result Settings</string>
   </property>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Result &amp;Title:</string>
    </property>
    <property name="buddy">
     <cstring>lineEdit</cstring>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>551</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string>Data Sources</string>
   </property>
   <widget class="QComboBox" name="dsTypeCombo">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>391</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Data Source &amp;Type:</string>
    </property>
    <property name="buddy">
     <cstring>dsTypeCombo</cstring>
    </property>
   </widget>
   <widget class="QListWidget" name="dsList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>411</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Data Source &amp;List:</string>
    </property>
    <property name="buddy">
     <cstring>dsList</cstring>
    </property>
   </widget>
   <widget class="QPushButton" name="dsAdd">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>100</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dsEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>160</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dsRemove">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>190</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dsDown">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>280</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dsUp">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>250</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Scale Units:</string>
    </property>
    <property name="buddy">
     <cstring>dsScaleUnitsCombo</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="dsScaleUnitsCombo">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>301</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="dsAddMulti">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>130</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Multiple...</string>
    </property>
   </widget>
   <widget class="QLabel" name="rbLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Layout:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbTime">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>360</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Time Based</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbRes">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>360</y>
      <width>251</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Residue Based</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>dsTypeCombo</tabstop>
  <tabstop>dsList</tabstop>
  <tabstop>dsAdd</tabstop>
  <tabstop>dsAddMulti</tabstop>
  <tabstop>dsEdit</tabstop>
  <tabstop>dsRemove</tabstop>
  <tabstop>dsUp</tabstop>
  <tabstop>dsDown</tabstop>
  <tabstop>dsScaleUnitsCombo</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>resultDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
